### PR TITLE
Feature a11y

### PR DESCRIPTION
**Pull request by kennethlarsen**
I'm not sure what your scope is for the UI Toolbar so feel free to decline this. 

This is the fork I personally use as it includes the tota11y toolbar used for checking accessibility issues in the markup. Tota11y can be used on its own but I think it's nice to have included in the UI Toolbar (might also make people actually test for accessibility).

### CODE REVIEW

**Comment by madsobel** [comment]
Awesome Kenneth!
Would it make sense to reference tota11y from NPM or a CDN such as https://cdnjs.com/libraries/tota11y
rather than checking in a file? Makes it a lot easier to upgrade in the future.

**Comment by kennethlarsen** [comment]
Good point - I started working on it before you pushed the package.json stuff. I'll update the PR soon.

**Comment by kennethlarsen** [comment]
Seems like their NPM module breaks our gulp build so I used the CDN for now.

**Comment by madsobel** [comment]
Cool. Can you remove the dist folder? I've added it to gitignore now and have travis to build and deloy it for us instead. Local versions of dist is meant only for development now.

File: js/tota11y.min.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/*!
```
Comment (grandorf) [inline js/tota11y.min.js]: Should we remove this file now?

Comment (kennethlarsen) [reply js/tota11y.min.js]: Yes, actually thought I did. Will update PR with it removed.

**Pull request merged**